package/kernel/linux/modules/netsupport.mk

```diff
@@ -1250,3 +1250,34 @@ define KernelPackage/netlink-diag/description
 endef
 
 $(eval $(call KernelPackage,netlink-diag))
+
+
+define KernelPackage/wireguard
+  SECTION:=kernel
+  CATEGORY:=Kernel modules
+  SUBMENU:=Network Support
+  TITLE:=WireGuard kernel module
+  KCONFIG:= \
+	  CONFIG_WIREGUARD \
+	  CONFIG_WIREGUARD_DEBUG=n
+  DEPENDS:= \
```

Comment:
swap the position of `KCONFIG` and `DEPENDS` is recommended.

Reply:
Okay, will do, can you explain why?

Reply:
however it's juat a habit, you may see other packages added here.
also, althouth we have AddDepends in `crypto.mk`, but using `DEPENDS:=` is more common.